modules/Core/src/main/java/org/terasology/logic/actions/TunnelAction.java

```diff
@@ -50,15 +57,25 @@
     private EntityManager entityManager;
 
     private Random random = new FastRandom();
+    private List<Optional<StaticSound>> explosionSounds = Lists.newArrayList();
```

Comment:
A list of optionals is strange. If an asset isn't there it should not be in the list or there should be actually an exception when the asset could not be found.